Source/Public/ZMUpdateEvent.swift

```diff
@@ -31,6 +31,37 @@ import WireUtilities
     case download
 }
 
+@objc public enum ZMParticipantsRemovedReason : Int16, CaseIterable {
```

Comment:
The ZM prefix is only necessary in objective c, so I'd suggest to drop it for the Swift type:

```suggestion
@objc(ZMParticipantsRemovedReason)
public enum ParticipantsRemovedReason : Int16, CaseIterable {
```